ui/desktop/electron-app/src/session/manager.js

```diff
@@ -0,0 +1,38 @@
+const Session = require('./index.js');
+
+class SessionManager {
+  #sessions;
+
+  constructor() {
+    this.#sessions = [];
```

Comment:
Isn't this equivalent to initializing it in the declaration above?

Reply:
Focused on maintaining a class pattern by defining private vars first, followed by constructors, getters, setters, and methods. I see your point.